modules/apps/portal-background-task/portal-background-task-service/src/main/java/com/liferay/portal/background/task/internal/BackgroundTaskManagerImpl.java

```diff
@@ -626,7 +626,7 @@ protected void activate(BundleContext bundleContext) {
 
 		BackgroundTaskQueuingMessageListener
 			backgroundTaskQueuingMessageListener =
-				new BackgroundTaskQueuingMessageListener(this);
+				new BackgroundTaskQueuingMessageListener(this, _lockManager);
```

Comment:
Hi @dantewang ,
For this kind of fix, please ask Leon to redo next time.
I do not like to keep an wrong commit (his original one) in the commit history. I will take it this time, but please do not do this next time.

Thanks.
Tina.